### DISCUSSION TITLE
Rich text editor

### DISCUSSION THREAD

**Discussion by proficientdesigners** [pocketbase/pocketbase#369] [Q&A]
Adding a rich text editor or markdown editor would make this app to be useful for a startup blog, any plans on that ?

**Comment by ganigeorgiev** [comment]
This was recently discussed in https://github.com/pocketbase/pocketbase/discussions/360.

I'll consider implementing it in a future version (you can subscribe to this issue - https://github.com/pocketbase/pocketbase/issues/370), but for now it is a low priority.
